modules/features/manatime_feature_timesheets/includes/timesheet.inc

```diff
@@ -101,6 +101,28 @@ class Timesheet {
   }
 
   /**
+   * Get projects blocked, without available tasks.
+   */
+  public function getBlockedProjects() {
```

Comment:
What's a blocked project? Please add comments to the function to make clear what's happening there.